### PR TITLE
CIV-7845 Evidence upload submission confirmation screen

### PR DESCRIPTION
**Pull request by jakiurmalik**
### JIRA link ###
https://tools.hmcts.net/jira/browse/CIV-7845

### Change description ###
Implemented new documents upload screen for evidence uploads.

image

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ x] No
```

### CODE REVIEW

**Review by m-meulendijks-v1 — APPROVED** [review]

**Review by kurt-uk — DISMISSED** [review]

**Review by hmcts-version1-ignacio — CHANGES_REQUESTED** [review]

File: src/main/views/features/caseProgression/documents-uploaded.njk [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+{% block pageTitle %}
+  {{ setPageTitle('PAGES.SUBMIT_CONFIRMATION.PAGE_TITLE') }}
+{% endblock %}
+
+{% set panelContent %}
+  {{ t('PAGES.EVIDENCE_DOCUMENTS_UPLOADED.CASE_REFERENCE_NUMBER')}}
+  <br><strong>{{ caseNumber }}</strong><br>
+{% endset %}
+
+{% from "../../macro/contact-us-for-help.njk" import contactUsForHelp %}
```
Comment (laarmada) [inline src/main/views/features/caseProgression/documents-uploaded.njk:17]: move at the bottom

File: src/main/views/features/caseProgression/documents-uploaded.njk [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+{% block content %}
+<div class="govuk-width-container govuk-!-padding-top-0">
+  <main class="govuk-main-wrapper govuk-!-padding-top-0" role="main">
+    <div class="govuk-grid-row govuk-!-margin-top-0">
+      <div class="govuk-grid-column-two-thirds govuk-!-margin-top-0">
```
Comment (laarmada) [inline src/main/views/features/caseProgression/documents-uploaded.njk:23]: {{ csrfProtection(csrf) }} is missing

**Review by laarmada — CHANGES_REQUESTED** [review]

File: src/main/views/features/caseProgression/documents-uploaded.njk
[the remark this answers is not in the document]
Comment (hmcts-version1-ignacio) [reply src/main/views/features/caseProgression/documents-uploaded.njk:1]: Done.

File: src/main/routes/features/caseProgression/documentsUploadedController.ts [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+const documentsUploadedController = Router();
+documentsUploadedController.get(CP_EVIDENCE_UPLOAD_SUBMISSION_URL, (async (req:Request, res:Response, next: NextFunction) => {
+  try {
+    const claimId = req.params.id;
+    const claim: Claim = await getCaseDataFromStore(claimId);
+    const uploadYourDocumentsUrl = UPLOAD_YOUR_DOCUMENTS_URL.replace(':id', claim.id);
+    const documentsTabUrl = DEFENDANT_SUMMARY_URL.replace(':id', claim.id)+'#documents';
+    const caseNumber = caseNumberPrettify(claimId);
+
+    if (claim && !claim.isEmpty()) {
```
[the remark this answers is not in the document]
Comment (hmcts-version1-ignacio) [reply src/main/routes/features/caseProgression/documentsUploadedController.ts:18]: Updated to "Something went wrong" message if the claim does not exist.

**Review by kurt-uk — APPROVED** [review]

**Review by hmcts-version1-ignacio — APPROVED** [review]

**Review by LeonardoPalmeiro — APPROVED** [review]

**Review by kurt-uk — APPROVED** [review]

**Review by hmcts-version1-ignacio — APPROVED** [review]

**Pull request merged**